src/test/java/app/daos/MovieDAOTest.java

```diff
@@ -129,40 +125,40 @@ void setUp() {
     }
 
     @Test
-    void getById() {
-        MovieDTO expected = movieDTOS.get(0);
+    void testGetById() {
+        MovieDTO expected = movieDTOs.get(0);
+
         MovieDTO actual = movieDAO.getById(expected.getId());
 
         Assertions.assertEquals(expected, actual);
     }
 
     @Test
-    void getAll() {
-        Set<MovieDTO> expected = new HashSet<>(movieDTOS);
+    void testGetAll() {
+        Set<MovieDTO> expected = new HashSet<>(movieDTOs);
         Set<MovieDTO> actual = movieDAO.getAll();
 
         Assertions.assertEquals(expected, actual);
     }
-
```

Comment:
Empty line between methods seems to have been removed. It should be added back in for better readability

Reply:
changeeddd